mobile_verifier/src/radio_threshold.rs

```diff
@@ -391,14 +390,15 @@ pub async fn delete(
     ingest_report: &InvalidatedRadioThresholdIngestReport,
     db: &mut Transaction<'_, Postgres>,
 ) -> Result<(), sqlx::Error> {
+    let cbsd_id: Option<String> = None;
     sqlx::query(
         r#"
             DELETE FROM radio_threshold
             WHERE hotspot_pubkey = $1 AND (cbsd_id is null or cbsd_id = $2)
         "#,
     )
     .bind(ingest_report.report.hotspot_pubkey.to_string())
-    .bind(ingest_report.report.cbsd_id.clone())
+    .bind(cbsd_id)
```

Comment:
should be able to just nix this binding altogether and change the query to _just_ be `cbsd_id is null` if that distinction is important going forward